packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -431,7 +431,13 @@ export interface IFluidParentContext
 	 */
 	readonly scope: FluidObject;
 
+	/**
+	 * @deprecated - This functionality has been removed.
```

Comment:
Deprecated these two for now. Didn't remove it because it would break legacy alpha APIs which is not allowed atm.